utils/podman/Makefile

```diff
@@ -93,10 +93,12 @@ endef
 define Package/podman/install
 	$(call GoPackage/Package/Install/Bin,$(1))
 	$(INSTALL_DIR) $(1)/etc/containers
+	chmod 0755 $(1)/etc/containers
 	$(INSTALL_CONF) $(DL_DIR)/default-policy.json-362f70b056 $(1)/etc/containers/policy.json
 	$(INSTALL_CONF) $(DL_DIR)/registries.fedora-da9a9c8778 $(1)/etc/containers/registries.conf
 	$(INSTALL_CONF) $(PKG_BUILD_DIR)/vendor/github.com/containers/storage/storage.conf $(1)/etc/containers/storage.conf
 	$(INSTALL_CONF) ./files/containers.conf $(1)/etc/containers/containers.conf
+	chmod 0644 $(1)/etc/containers/*
```

Comment:
Please you use `$(INSTALL_DATA)` instead then, that will set `0644` permissions, see
https://github.com/openwrt/openwrt/blob/master/rules.mk#L285

Reply:
Fixed. Thank you.